official/keras_application_models/benchmark_main.py

```diff
@@ -54,6 +58,10 @@ def run_keras_model_benchmark(_):
     raise AssertionError("The --model command line argument should "
                          "be a key in the `MODELS` dictionary.")
 
+  # Check if eager mode is enabled
```

Comment:
Can we add the caveats you mentioned in the PR message to the comments somewhere too? Maybe to the eager flag help.